data/Makefile.am

```diff
@@ -33,8 +36,16 @@ desktop_DATA =						\
 	setup-shell.desktop				\
 	$(NULL)
 
+demovideodir = $(datadir)/eos-demo-mode
+demovideo_DATA = \
+	eos-demo-video.ogv \
+	eos-demo-video.desktop \
```

Comment:
One less moving piece, and it just feels cleaner to ship the file where it's supposed to go from the OSTree instead of copying it over every time.